plugins/__init__.py

```diff
@@ -582,10 +582,10 @@ def name_matches(name):
     allrelations = [r for e in mytypes.values() for r in e.get_attributes().values() if isinstance(r, RelationAttribute)]
     if not relations_escape:
         allrelations = [r for r in allrelations if r.get_type().get_full_name() in mytypes]
-    paired = set()
+    paired = dict()
```

Comment:
Could you add a comment as to why you use a dict? It's obvious now but it might not be when next we look at this module.